packages/@aws-cdk/toolkit-lib/lib/toolkit/toolkit.ts

```diff
@@ -153,6 +156,69 @@ export class Toolkit extends CloudAssemblySourceBuilder implements AsyncDisposab
     };
   }
 
+  /**
+   * Bootstrap Action
+   */
+  public async bootstrap(cx: ICloudAssemblySource, userEnvironmentSpecs: string[], options: BootstrapOptions = {}): Promise<void> {
```

Comment:
Finally (and this can be a follow up PR). What would be a useful return value here?
```suggestion
  public async bootstrap(cx: ICloudAssemblySource, userEnvironmentSpecs: string[], options: BootstrapOptions = {}): Promise<???> {
```